services/docs.ubuntu.com.yaml

```diff
@@ -63,6 +78,39 @@ spec:
 
 ---
 
+kind: Deployment
+apiVersion: apps/v1beta1
+metadata:
+  name: docs-ubuntu-com-core
+spec:
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        app: docs.ubuntu.com-core
+    spec:
+      containers:
+        - name: docs-ubuntu-com-core
+          image: prod-comms.docker-registry.canonical.com/docs.ubuntu.com-core:${TAG_TO_DEPLOY_core}
```

Comment:
`TAG_TO_DEPLOY_core` core lower case?

Reply:
Was following the way `TAG_TO_DEPLOY_phone` was written

Reply:
do we want to keep it that way?
It seems strange to have this var like this...

Reply:
It's a fair point. I don't mind about the casing here. But if we change it we need to update the Jenkins jobs.

Reply:
Let's put every global var upper case!

Reply:
OK, I added a commit